src/PrivateDns/setenv_test.cmd

```diff
@@ -0,0 +1,12 @@
+set SUBSCRIPTIONID=%1
+set TESTENV=%2
```

Comment:
This also seems unnecessary - is there a reason you need this and our current documentation is not sufficient?

Reply:
deleted it.